src/App.js

```diff
@@ -1,25 +1,8 @@
-import logo from './logo.svg';
-import './App.css';
+import calculator from './components/Calculator';
 
```

Comment:
- Kindly delete `logo.svg` and other unused files so as to keep your project structure clean.